### DISCUSSION TITLE
Load sql view without database name

### DISCUSSION THREAD

**Discussion by ccastilla** [dbeaver/dbeaver#12626] [Q&A]
Hello, Is it possible to open the sql code of a mysql view without containing the name of the database that contains it.

```
create view myView
select 
field1, field2
from mytable
```
and not...

```
create view myView
select 
field1, field2
from mydatabase.mytable
```

thanks!

**Comment by flambchap** [comment]
Hello. It is absolutely possible. Be sure though to set the right database in the active catalog selector.
![Screenshot 2021-05-27 174010](https://user-images.githubusercontent.com/26031301/119846322-b318de00-bf12-11eb-8489-9f92958c0f91.png)

**Reply by ccastilla** [reply]
Excuse me, where can I configure it?

![image](https://user-images.githubusercontent.com/37351819/119860576-14718a80-bedc-11eb-974f-8c06e8b337b4.png)

**Reply by flambchap** [reply]
Oh, I'm sorry, I misunderstood your first message. Unfortunately, there is no way to do that in DBeaver right now. You can open a feature request on our issue tracker if you want it to be implemented.

**Reply by ccastilla** [reply]
ok i understand, thank you very much for your time!
